digitalocean/spaces/resource_spaces_bucket.go

```diff
@@ -255,6 +255,11 @@ func resourceDigitalOceanBucketCreate(ctx context.Context, d *schema.ResourceDat
 
 		return nil
 	})
+
+	if err != nil {
+		return diag.Errorf("Error waiting for Spaces bucket: %s", err)
```

Comment:
Great job! :clap:  :clap:  :clap:
Lets clarify the error message a bit because the error relates specifically to the `svc.HeadBucket` and `resource.RetryContext` is a general wrapper. `svc.HeadBucket` checks the bucket's availability. 
Thanks for your effort!

```suggestion
		return diag.Errorf("Failed to check availability of Spaces bucket %s: %s", name, err)
```

Reply:
Thanks! Done.